### DISCUSSION TITLE
Search not working after upgrade to 2.67.5 version

### DISCUSSION THREAD

**Discussion by bbsparkrock** [dotnet/docfx#9006] [General]
After upgrading to version 2.67.5 from 2.59.3, when I try to perform a search, nothing happens. Please assist.

The browser console displays the following:
TypeError: Cannot read properties of undefined (reading 'split')
    at highlightKeywords (docfx.js:471:26)
    at enableSearch (docfx.js:275:7)
    at HTMLDocument.<anonymous> (docfx.js:25:3)
    at ee (jquery.js:3830:29)
    at Ge (jquery.js:3898:12)
enableSearch @ docfx.js:281
(anonymous) @ docfx.js:25
ee @ jquery.js:3830
Ge @ jquery.js:3898
setTimeout (async)
(anonymous) @ jquery.js:3936
h @ jquery.js:3564
fireWith @ jquery.js:3694
fire @ jquery.js:3702
h @ jquery.js:3564
fireWith @ jquery.js:3694
ready @ jquery.js:4174
Xe @ jquery.js:4184

FYI. The website is hosted on an Azure website.

**Comment by KalleOlaviNiemitalo** [comment]
Which web browser are you using?  There have been compatibility problems with Firefox ESR.

**Comment by bbsparkrock** [comment]
I have tested in Chrome and Edge.

**Comment by filzrev** [comment]
From log stack trace. It seems error is occurred at `highlightKeywords` method.(It's is called on page load). 
https://github.com/dotnet/docfx/blob/main/templates/default/styles/docfx.js#L237

```
TypeError: Cannot read properties of undefined (reading 'split')
at highlightKeywords (docfx.js:471:26)
at enableSearch (docfx.js:275:7)
at HTMLDocument. (docfx.js:25:3)
```

But `docfx.js:471` line number is not matched `2.67.5` nor  `2.59.3` version's `docfx.json`.
It seems `docfx.js` that deployed to your Azure website is not original version.
Instead, some modified version is used?
